### PR TITLE
Add blade admin flag

### PR DESCRIPTION
**Pull request by goran-ethernal**
# Description

This PR adds a new flag to `genesis` command which represents an address of the owner/admin of `NativeERC20` token on `blade`, and owner/admin of `StakeManager` contract which can whitelist new validators.

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] Hotfix (change that solves an urgent issue, and requires immediate attention)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Breaking changes

Please complete this section if any breaking changes have been made, otherwise delete it

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [ ] I have added the relevant labels
- [ ] I have updated the official documentation
- [ ] I have added sufficient documentation in code

## Testing

- [ ] I have tested this code with the official test suite
- [ ] I have tested this code manually

### CODE REVIEW

File: consensus/polybft/polybft_config.go [hunk 1]
```diff
@@ -77,11 +76,15 @@ type PolyBFTConfig struct {
...
 	// at which block tracker polls for blocks on a rootchain
-	BlockTrackerPollInterval common.Duration `json:"blockTrackerPollInterval,omitempty"`
+	BlockTrackerPollInterval common.Duration `json:"blockTrackerPollInterval"`
 
 	// ProxyContractsAdmin is the address that will have the privilege to change both the proxy
 	// implementation address and the admin
-	ProxyContractsAdmin types.Address `json:"proxyContractsAdmin,omitempty"`
+	ProxyContractsAdmin types.Address `json:"proxyContractsAdmin"`
+
+	// BladeAdminAddress is the address that will be the owner of the NativeERC20 mintable token,
```
Comment (Stefan-Ethernal) [inline consensus/polybft/polybft_config.go]:
```suggestion
	// BladeAdmin is the address that will be the owner of the NativeERC20 token,
```

File: consensus/polybft/polybft_config.go [hunk 2]
```diff
@@ -77,11 +76,15 @@ type PolyBFTConfig struct {
...
+	// and StakeManager contract which handles validator actions like whitelist
```
Comment (Stefan-Ethernal) [inline consensus/polybft/polybft_config.go]:
```suggestion
	// and StakeManager contract which manages validators
```

**Review by Stefan-Ethernal — APPROVED** [review]

File: consensus/polybft/polybft_config.go (hunk 1 above)
Comment (goran-ethernal) [reply consensus/polybft/polybft_config.go]: Fix: https://github.com/Ethernal-Tech/blade/pull/24/commits/0d9f4c50161d7c5f3ba40d3f2736f179f91b99cb

File: consensus/polybft/polybft_config.go (hunk 2 above)
Comment (goran-ethernal) [reply consensus/polybft/polybft_config.go]: Fix: https://github.com/Ethernal-Tech/blade/pull/24/commits/0d9f4c50161d7c5f3ba40d3f2736f179f91b99cb

**Pull request merged**